scripts/proposals/simulateExistingProposal.ts

```diff
@@ -0,0 +1,28 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {getPolygonContracts, getForkParams} from '../../shared/Forking';
+import {createAndExecuteProposal} from '../../shared/Governance';
+
+export async function simulateExistingProposal(proposalId: string, hre: HardhatRuntimeEnvironment) {
+  const [user] = await hre.ethers.getSigners();
+  const deployment = getPolygonContracts(user);
+
+  // attempt mainnet forking
+  await hre.network.provider.request({
+    method: 'hardhat_reset',
+    params: [getForkParams()],
+  });
+
+  const proposalActions = await deployment.governor.getActions(proposalId);
+  let valuesArray = [...proposalActions[1]].map((value) => value.toString());
```

Comment:
Isn't `[...arr]` the same as just `arr`? (except creating a copy but `map` creates a copy anyway)